src/OrchardCore/OrchardCore.DisplayManagement/Razor/RazorPage.cs

```diff
@@ -404,6 +446,16 @@ public ISite Site
                 return _site;
             }
         }
+
+        private async Task<bool> IsFeatureEnabledAsync(string featureId)
+        {
+            EnsureShellFeaturesManager();
+
+            var feature = (await _shellFeaturesManager.GetEnabledFeaturesAsync())
+                .FirstOrDefault(f => f.Id == featureId);
+
```

Comment:
You need to also look for `GetAlwaysEnabledFeaturesAsync()` here.

Reply:
I presume it should get all enabled features, I saw similar call in features admin controll